JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -363,5 +363,16 @@ module Microsoft.ApplicationInsights {
             
             return true;
         }
+
+        public static getAbsoluteUrl = (function () {
+            var a;
+
+            return function (url) {
+                if (!a) a = document.createElement('a');
```

Comment:
I think you're right - it does affect minification (as a result we'll have 'document' instead of some sort of minified string out of 'util.document').